cli/internal/terraform/terraform/qemu/modules/instance_group/main.tf

```diff
@@ -22,26 +22,24 @@ resource "libvirt_domain" "instance_group" {
     backend_type    = "emulator"
     backend_version = "2.0"
   }
-  disk = [
-    {
-      volume_id = element(libvirt_volume.boot_volume.*.id, count.index)
-      scsi : true,
-      // fix for https://github.com/dmacvicar/terraform-provider-libvirt/issues/728
-      block_device : null,
-      file : null,
-      url : null,
-      wwn : null
-    },
-    {
-      volume_id = element(libvirt_volume.state_volume.*.id, count.index)
-      // fix for https://github.com/dmacvicar/terraform-provider-libvirt/issues/728
-      block_device : null,
-      file : null,
-      scsi : null,
-      url : null,
-      wwn : null
-    },
-  ]
+  disk {
+    volume_id = element(libvirt_volume.boot_volume.*.id, count.index)
+    scsi      = true
+    // fix for https://github.com/dmacvicar/terraform-provider-libvirt/issues/728
+    block_device = null
+    file         = null
+    url          = null
```

Comment:
Must these values be explicitly set to null?

Reply:
The linked issues seems like this is still not fixed.
However, manual testing works with the nulled types removed